bot/utils/decorators.py

```diff
@@ -199,23 +199,38 @@ def predicate(ctx: Context) -> bool:
         kwargs = default_kwargs.copy()
         allow_dms = False
 
-        # Update kwargs based on override
-        if hasattr(ctx.command.callback, "override"):
+        # determine which command's overrides we will use
+        # as we have groups, we want to ensure that group commands inherit from the parent
```

Comment:
The way this comment is unevenly split the way it is... honestly just irks me 😅.

Not a big deal, but could quickly be changed when you're making other changes.

Reply:
```suggestion
        # Determine which command's overrides we will use. Group commands will
        # inherit from their parents if they don't define their own overrides
```